src/main/java/seedu/billboard/commons/core/date/DateInterval.java

```diff
@@ -17,6 +17,9 @@
     MONTH("month", Period.ofMonths(1), TemporalAdjusters.firstDayOfMonth()),
     YEAR("year", Period.ofYears(1), TemporalAdjusters.firstDayOfYear());
 
+    public static final String MESSAGE_CONSTRAINTS =
```

Comment:
This constant seems to not be used at all

Reply:
It is used in the parseInterval() method in ParserUtil